libs/utils/address.go

```diff
@@ -30,11 +31,14 @@ func ValidateAddr(addr string) (string, error) {
 	}
 
 	if ip := net.ParseIP(addr); ip == nil {
-		_, err = net.LookupHost(addr)
+		addrs, err := net.LookupHost(addr)
 		if err != nil {
 			return addr, fmt.Errorf("could not resolve hostname or ip: %w", err)
 		}
+		if len(addrs) == 0 {
+			return addr, errors.New("no IP addresses found for DNS record")
```

Comment:
optional nit: would be nice to include actual addresses in the error

Reply:
Added addr, thanks for suggestion!